sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdResponse.java

```diff
@@ -135,6 +139,24 @@ public Long getTransportRequestId() {
         return this.getHeader(RntbdResponseHeader.TransportRequestID);
     }
 
+    @JsonIgnore
+    public Instant getDecodeStartTime() {
+        return decodeStartTime;
+    }
+
+    @JsonIgnore
+    public void setDecodeStartTime(Instant decodeStartTime) {
+        this.decodeStartTime = decodeStartTime;
+    }
+
+    public Instant getDecodeEndTime() {
+        return decodeEndTime;
+    }
+
+    public void setDecodeEndTime(Instant decodeEndTime) {
```

Comment:
Is this setter missing `@JsonIgnore` annotation?

Reply:
thanks, good catch, updated